tests/unit/data_encoding/test_encoding_decoding.py

```diff
@@ -0,0 +1,42 @@
+"""
+To Test the module message_encoding_decoding.py
+"""
+
+from modules import data_encoding
+
+from modules import position_global
+
+
+def test_encoding_decoding() -> None:
+    """
+    Function to test encoding
+    """
+
+    # Step 1: Create a worker_name and PositionGlobal object
+    worker_name = "communications_worker"  # =3 in Worker_Enum
+    success, original_position = position_global.PositionGlobal.create(
+        latitude=34.24902422, longitude=84.6233434, altitude=27.4343424
+    )
+    if not success:
+        return
```

Comment:
You should use assert here because the test still fails if you can't create a PositionGlobal.